pair-finder/pair_finder/pipeline/steps/get_jobs_from_travis_api.py

```diff
@@ -72,6 +73,17 @@ def process(self, data: Any, context: dict) -> Optional[Any]:
         #   WHERE j.repo_id = "<repo_id>"
         jobs = []
         for build in build_list:
+
+            try:
+                # build['finished_at'] returns an ISO 8601 time representation. Ex - 2015-07-13T12:40:51Z
+                # while context['..']['_updated'] returns an ISODate object formatted as: Tue, 28 Apr 2015 00:00:00 GMT.
+                # We must reformat the time ISO time representation to be formatted similarly so we can compare.
+                t = dp.parse(build['finished_at'])
+                build_formatted_date = t.strftime('%a, %d %b %Y %H:%M:%S GMT')
```

Comment:
What type does `strftime()` return? Is it a string?

Reply:
Yes it returns a string.